rewards/classes/Snapshot.py

```diff
@@ -16,8 +16,9 @@
 
 
 class Snapshot:
-    def __init__(self, token, balances, ratio=1, type="none"):
+    def __init__(self, token, balances, ratio=1, type="none", chain="eth"):
```

Comment:
is there reason why chain wouldn't be Network here? needed differently for some api call?

Reply:
Yeah, that makes sense.I just noticed this pr also includes the blacklist stuff i was working on, this stuff should probably be reverted and addressed in a different pr